src/sprites/player.js

```diff
@@ -0,0 +1,17 @@
+import Phaser from 'phaser';
+
+export default class extends Phaser.Sprite {
+    constructor({ game, x, y, asset }) {
+        super(game, x, y, asset);
```

Comment:
Just pointing this out: Does `super()` take 4 function parameters `game`, `x`, `y`, `asset`, _or_ does it accept a single object like your own `constructor()` ?

e.g. `super(game, x, y, asset);` vs `super({ game, x, y, asset });`